tests/unit/amo/reducers/test_categories.js

```diff
@@ -286,4 +292,147 @@ describe(__filename, () => {
       expect(state.loading).toEqual(true);
     });
   });
+
+  describe('getCategories', () => {
+    let state;
```

Comment:
not sure why this isn't just declared as a `const` further down when first used, like `results` and `addon` are.